CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -244,31 +251,55 @@ private void CopyTo(ByteBuffer buffer) {
     }     
     
     
-    private static int LoadShaders(CVKDevice cvkDevice) {
+    private static int LoadShaders(CVKDevice cvkDevice) {       
         int ret = VK_SUCCESS;
 
         try{
-            SPIRV vertShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "PassThru.vs", VERTEX_SHADER);
-            SPIRV geomShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "PassThruLine.gs", GEOMETRY_SHADER);
-            SPIRV fragShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "PassThru.fs", FRAGMENT_SHADER);
+            if (vertexShaderSPIRV == null) {
+                vertexShaderSPIRV = compileShaderFile(CVKShaderPlaceHolder.class, "PassThru.vs", VERTEX_SHADER);
```

Comment:
There should still be a null check, just return an error code instead.

Reply:
It does a null check and returns an error code and logs message in the except block

Reply:
I don't see any null check.

Reply:
added null check and error code in